_docs/_latest/configure/scanner/descriptions.md

```diff
@@ -32,7 +32,7 @@ all of the VM instances in your environment and determines if any VMs
 with external IP addresses are on a specific blacklist you've configured.
 
 For examples of how to define scanner rules, see the
-[`blacklist_rules.yaml`](https://github.com/forseti-security/forseti-security/blob/master/samples/scanner/scanners/blacklist_rules.yaml) rule file.
+[`blacklist_rules.yaml`](https://github.com/forseti-security/terraform-google-forseti/blob/master/modules/rules/templates/rules/blacklist_rules.yaml) rule file.
```

Comment:
I'm curious why the rules paths were changed. The default rules were recently moved to the samples directory of the FS repo. I think these links should remain pointing to the sample dir.

Reply:
Updated links to point to FS repo samples dir.